rainforest/rainforest.go

```diff
@@ -57,7 +57,12 @@ func NewClient(token string, debug bool) *Client {
 		baseURL, _ = url.Parse(currentBaseURL)
 	}
 
-	return &Client{client: http.DefaultClient, BaseURL: baseURL, ClientToken: token, LastResponseHeaders: http.Header{}, DebugFlag: debug}
+	return &Client{client: http.DefaultClient, BaseURL: baseURL, clientToken: token, LastResponseHeaders: http.Header{}, DebugFlag: debug}
+}
+
+// ClientToken returns the API authentication token for the client.
+func (c *Client) ClientToken() string {
```

Comment:
So that it could be overwritten in tests. But it breaks backward-compatibility I realized, maybe not the best idea after all... (Or we should just change `Client` to be an interface, which would make testing in other modules quite a bit easier.)